pkg/cmd/roachtest/log.go

```diff
@@ -76,8 +90,10 @@ func (cfg *loggerConfig) newLogger(path string) (*logger, error) {
 	if path == "" {
 		// Log to stdout/stderr if there is no artifacts directory.
 		return &logger{
-			stdout: os.Stdout,
-			stderr: os.Stderr,
+			stdout:  os.Stdout,
+			stderr:  os.Stderr,
+			stdoutL: log.New(os.Stdout, cfg.prefix, logFlags),
+			stderrL: log.New(os.Stdout, cfg.prefix, logFlags),
```

Comment:
If `os.Stdout == os.Stderr`, `stdoutL` and `stderrL` will pointer to different `log.Loggers`. Isn't that a problem? Any synchronization they perform internally will be worthless.